include/dsn/cpp/auto_codes.h

```diff
@@ -167,56 +167,6 @@ namespace dsn
 #endif
     };
 
-    class atom_int
```

Comment:
This is previously added for having atom<int> in .thrift. Make sure it still works and we don't need to handle it manually.